app/views/trainees/funding/training_initiatives/edit.html.erb

```diff
@@ -13,7 +13,7 @@
       <h1 class="govuk-heading-l"><%= t("components.page_titles.trainees.funding.training_initiatives.edit") %><h1>
       <p class="govuk-body"><%= t("views.forms.funding.training_initiatives.description_html") %></p>
 
-        <%= f.govuk_radio_buttons_fieldset(:training_initiative, legend: { text: t("views.forms.funding.training_initiatives.#{training_initiative_options(@trainee).count == 2 ? 'title_two' : 'title'}"), size: "m" }) do %>
+        <%= f.govuk_radio_buttons_fieldset(:training_initiative, legend: { text: t("views.forms.funding.training_initiatives.#{training_initiative_options(@trainee).count == 2 ? 'title_two' : 'title'}"), size: "m" }, classes: "training-initiatives") do %>
```

Comment:
Do we need to add this class? Can't we get away with just using `.govuk-radios` in the siteprism thing? I don't like adding stuff to the page just so we can select it in the tests.

Reply:
All sorted